Core/Object Arts/Dolphin/Base/Collection.cls

```diff
@@ -323,6 +323,12 @@ errorNotKeyed
 
 	^self error: ('<1p>s do not respond to keyed accessing messages' expandMacrosWith: self class)!
 
+expandMacrosIn: aString
```

Comment:
John, this needs to be pushed down to SequenceableCollection, as the String>>expandMacrosWithArguments: method expects an ordered collection with integer keys (it says Array, but any sequence will do). I suppose you could theoretically pass it a Dictionary with integer keys, but that would be a bit perverse. It definitely won't work for Sets, etc.
If you are not in a position to update it today, I can merge and fix later.

Reply:
Yes, I won't be able to address this today so if you could merge and fix that would be great - thanks.